sdk/communication/Azure.Communication.CallAutomation/tests/CallAutomationClients/CallAutomationClientAutomatedLiveTests.cs

```diff
@@ -0,0 +1,149 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Azure.Communication.CallAutomation;
+using Azure.Communication.CallAutomation.Tests.Infrastructure;
+using Azure.Communication.Identity;
+using Azure.Core;
+using Azure.Core.TestFramework;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Azure.Amqp.Framing;
+using NUnit.Framework;
+using NUnit.Framework.Internal;
+
+namespace Azure.Communication.CallAutomation.Tests.CallAutomationClients
+{
+    internal class CallAutomationClientAutomatedLiveTests : CallAutomationClientAutomatedLiveTestsBase
+    {
+        public CallAutomationClientAutomatedLiveTests(bool isAsync) : base(isAsync)
+        {
+        }
+
+        [RecordedTest]
```

Comment:
curious, what does this [RecordedTest] do?

Reply:
Similar to test - but this allows it to be re-recorded when recording is not present. Description says:

This attribute replaces the [Test] attribute and will dynamically re-record recorded tests on failure when run in Playback mode.
Tests that are re-recorded will complete with a error status and indicate that copying the updated recording to SessionRecords is needed.